content/docs/pipelines/sdk/enviroment_variables.md

```diff
@@ -0,0 +1,72 @@
++++
+title = "Using enviroment varibales in piplines"
+description = "How to set and use enviroment variables in kubeflow piplines"
++++
+
+This page describes how to create a enviroment varaibale for Kubeflow Pipelines 
+and how they work in a pipline. 
+
+## Before you start
+
+### Initial setup
+Before starting with this tutorial the following things have to be in place: 
+- [Kubeflow set up on kubernetes](https://www.kubeflow.org/docs/gke/deploy/deploy-cli/)
+- Jupyter nooteboks set up on you kubeflow deployment
+
+
+## Using enviroment variables 
+
+Below is a example python lightweight component where we read a enviroment
+variable and the logging it out as a string. 
+
+We are going to use jupyter ligthweight componetes you can find more about them 
+[here](https://www.kubeflow.org/docs/pipelines/sdk/lightweight-python-components/).
+
+To build a component, define a stand-alone Python function and then call 
+`kfp.components.func_to_container_op(func)` to convert the function to a 
+component that can be used in a pipeline. Below is one small function logging 
```

Comment:
Replace the following:

Below is one small function logging out the enviroment variable.

With the following:

The following function gets an environment variable and writes it to the log.